### PR TITLE
added docopt to improve cli

### PR DESCRIPTION
**Pull request by giritheja**
added docopt to improve cli

### CODE REVIEW

**Comment by kishorbhat** [comment]
Thanks for contributing!

Before we merge this, however, a few things that need to be fixed:
- you can just install `docopt`: `$ pip install docopt`. There's no need to include the library code in a separate file in the directory.
- in `generator.py`: instead of `size`, having separate flags for width and height would be nice.
- having `docopt` support in `mazerunner.py` would also be nice, as mentioned in the issue.

**Comment by giritheja** [comment]
updated.

**Comment by kishorbhat** [comment]
One last thing: can you update `README.md` to reflect the change in usage of `generator` and `mazerunner`? Thanks!

**Comment by giritheja** [comment]
Updated.

File: README.rst [hunk 1]
```diff
@@ -7,11 +7,15 @@ Game made for the `Escape the Trolls <https://www.reddit.com/r/dailyprogrammer/c
 =======
 Running
 =======
+If you are running mazerunner for the first time. You need to install docopt.
 
+.. code:: bash
+
+    $ pip intall docopt
```
Comment (kishorbhat) [inline README.rst]: install?

Comment (giritheja) [reply README.rst]: My Bad. Updated.

**Pull request merged**